### PR TITLE
Don't attempt to install packages for Python new

### PR DESCRIPTION
**Pull request by swgillespie**
Global installation of packages is almost always not what a user will
want when running 'pulumi new'. This commit instead prints out the
commands that a user should run in order to create a new virtualenv and
install the required Pulumi packages within it.

### CODE REVIEW

**Comment by swgillespie** [comment]
Fixes part of https://github.com/pulumi/pulumi/issues/2201. Demo of it in action:

[asciicast](https://asciinema.org/a/9FeSx6gKyRUx9apixOa9oC351)

File: cmd/new.go [hunk 1]
```diff
@@ -524,36 +524,72 @@ func runUpOrPrintNextSteps(
...
+
+	if len(commands) == 0 { // No additional commands need to be run.
+		deployMsg := "To deploy it, run 'pulumi up'."
 		deployMsg = colors.Highlight(deployMsg, "pulumi up", colors.BrightBlue+colors.Underline+colors.Bold)
 		fmt.Println(opts.Display.Color.Colorize(deployMsg))
+		return
 	}
 
-	return nil
+	// One or more addditional commands needs to be run.
```
Comment (justinvp) [inline cmd/new.go]: addditional => additional

File: cmd/new.go [hunk 2]
```diff
@@ -524,36 +524,72 @@ func runUpOrPrintNextSteps(
...
+	}
 
-			cd = fmt.Sprintf("cd %s", cd)
+	return nil
+}
 
-			deployMsg = "To deploy it, '" + cd + "' and then run 'pulumi up'."
-			deployMsg = colors.Highlight(deployMsg, cd, colors.BrightBlue+colors.Underline+colors.Bold)
+// printNextSteps prints out a series of commands that the user needs to run before their stack is able to be updated.
+func printNextSteps(opts backend.UpdateOptions, originalCwd, cwd string, isGo, isPython bool) {
```
Comment (justinvp) [inline cmd/new.go]: Is `isGo` needed?

File: cmd/new.go (hunk 1 above)
Comment (swgillespie) [reply cmd/new.go]: yep, good catch.

File: cmd/new.go (hunk 2 above)
Comment (swgillespie) [reply cmd/new.go]: I guess it isn't - I'll get rid of it.

**Comment by lukehoban** [comment]
> Maybe we should simplify things and only show the pulumi new command for the time being?

@justinvp Honestly I think this makes sense for now - the E2E for `pulumi up <url>` doesn't feel like it's polished enough yet to advertise here.  And especially for `+ New Project` workflow I'm not sure it's really what we want to promote.  (It will be more important for the "installable apps" use cases).

**Comment by swgillespie** [comment]
Thanks!

> The difference in how we render pulumi up vs. the other instructions feels a little strange after this change. I wonder if we should either style it the same (green)? Especially since the underline and blue makes it look like a hyperlink which it isn't (not sure the original goal of that styling).

Yeah, it's a little funny - I can fix this in a follow-up PR if you want.

**Pull request merged**